io.catenax.message_header/1.0.0/MessageHeaderAspect.ttl

```diff
@@ -0,0 +1,126 @@
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.0.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.0.0#> .
+@prefix samm-e: <urn:samm:org.eclipse.esmf.samm:entity:2.0.0#> .
+@prefix unit: <urn:samm:org.eclipse.esmf.samm:unit:2.0.0#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix : <urn:samm:io.catenax.message_header:1.0.0#> .
+
+:MessageHeaderAspect a samm:Aspect ;
+   samm:preferredName "Message Header Aspect"@en ;
+   samm:description "Aspect model describing the message header which must be included in every message"@en ;
+   samm:properties ( :header ) ;
+   samm:operations ( ) ;
+   samm:events ( ) .
+
+:header a samm:Property ;
+   samm:preferredName "header"@en ;
+   samm:description "Contains information common to all messages"@en ;
+   samm:characteristic :HeaderCharacteristic .
+
+:HeaderCharacteristic a samm:Characteristic ;
+   samm:preferredName "Header Characteristic"@en ;
+   samm:description "Characteristic describing the common shared aspect Message Header"@en ;
+   samm:dataType :MessageHeader .
+
+:MessageHeader a samm:Entity ;
+   samm:preferredName "Message Header"@en ;
+   samm:description "The header that must be included in every message."@en ;
+   samm:properties ( :messageId :context :sentDateTime :senderBpn :receiverBpn [ samm:property :expectedResponseBy; samm:optional true ] [ samm:property :relatedMessageId; samm:optional true ] :version ) .
+
+:messageId a samm:Property ;
+   samm:preferredName "Message ID"@en ;
+   samm:description """Unique ID identifying the message.
+
```

Comment:
is the blank line on purpose? Depending on who the data is processed (eg in some implementation), it might look strange to have those blanks in it.